packages/concerto-core/lib/introspect/illegalmodelexception.js

```diff
@@ -14,7 +14,8 @@
 
 'use strict';
 
-const { BaseFileException } = require('@accordproject/concerto-util');
+// const { BaseFileException,ErrorCodes } = require('@accordproject/concerto-util');
```

Comment:
Could you please remove the commented out code?

Reply:
Yes!! Have removed.